distributed/comm/tcp.py

```diff
@@ -433,7 +433,9 @@ def _handle_stream(self, stream, address):
                      address, self.contact_address)
         local_address = self.prefix + get_stream_address(stream)
         comm = self.comm_class(stream, local_address, address, self.deserialize)
-        self.comm_handler(comm)
```

Comment:
Can we always `yield` here?

Reply:
Yes, probably... IIUC, for a regular function, `yield self.comm_handler(comm)` would just yield the concrete return value, which should be harmless.